libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/components/Divider.kt

```diff
@@ -35,7 +35,7 @@ fun Divider(
     thickness: Dp = ElementDividerDefaults.thickness,
     color: Color = DividerDefaults.color,
 ) {
-    androidx.compose.material3.Divider(
+    androidx.compose.material3.HorizontalDivider(
```

Comment:
Should we change the outer composable name too?

Reply:
Done in https://github.com/vector-im/element-x-android/pull/1011/commits/cbad2146f9b7dcebbecc90b143def2d56a32d064